index.html

```diff
@@ -115,19 +134,29 @@ <h3>Multi-Post Stories</h3>
                 <li class="padding-right">Ruby</li>
             </ul>
             <button type="button" class="project-button">See Project</button>
+
+            </div>            
+        
         </section> 
     </section>
     <section class="form-container">
-        <h3>CONTACT US</h3>
+        <h3 class="mobH">CONTACT US</h3>
+        <h3 class="deskH">Let`s chat!</h3>
         <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
         <form action="https://formspree.io/f/xdoylybq" method="POST">
             <label for="name"></label>
             <input type="text" id="name" name="Full name" placeholder="Full Name" required>
-            <label for="mail"></label>
+            <div class="form-box">
+                <label for="fname"></label>
+                <input type="text" id="fname" name="first name" placeholder="First name" required>
+                <label for="fname"></label>
+                <input type="text" id="lname" name="last name" placeholder="Last name" required>
+                <label for="mail"></label>
+            </div>
             <input type="email" id="mail" name="_replyto" placeholder="jeremy@gmail.com"  required>
             <label for="msg"></label>
             <textarea name="message" id="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
-            <button class="project-button" type="submit">See Project</button>
+            <button class="project-button" type="submit">Submit</button>
         </form>
     </section>
     </main>
```

Comment:
- [ ] The hr divider before the footer section should cover the width of the entire page as shown on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10).
- [ ] Your email should share the same row as the social media icon links as shown in the design.